### PR TITLE
Make alert header bold

### PR DESCRIPTION
**Pull request by ssandbekkhaug**
### Description

Bold matches [USWDS alerts](https://designsystem.digital.gov/components/alert/)

## Release notes

n/a

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | under-development | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers)
- [x] Added an additional reviewer from outside your organization as FYI (if the primary reviewer is in the same organization as you)
- [x] Created unit and/or browser tests which fail without the change (if possible)
- [x] Performed manual testing (Chrome and Firefox if it includes front-end changes)
- [x] Extended the README / documentation, if necessary

#### User visible changes

- [x] Followed steps to [internationalize](https://github.com/civiform/civiform/wiki/Internationalization-%28i18n%29#application-strings) any new strings
  - [x] Added context strings to new [messages](https://github.com/civiform/civiform/blob/main/server/conf/i18n/messages)
  - [x] Didn't use a message in applicant facing code that isn't translated yet (unless behind a flag)
- [x] Wrote browser tests using the [validateAccessibility](https://sourcegraph.com/github.com/civiform/civiform/-/blob/browser-test/src/support/index.ts?L437:14&subtree=true) method
- [x] Tested on mobile view. See [mobile device mode](https://developer.chrome.com/docs/devtools/device-mode/)
- [x] Manually tested at 200% size
- [x] Manually evaluated tab order

### Instructions for manual testing

n/a

### Issue(s) this completes

n/a

### CODE REVIEW

**Comment by ssandbekkhaug** [comment]
* **#8207** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8207?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ssandbekkhaug and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss [hunk 1]
```diff
@@ -26,6 +26,10 @@ See the radio-option below as an example.
   margin: 0 auto;
 }
 
+.usa-alert .usa-alert__heading {
```
Comment (swatkat1) [inline server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss:29]: Non-blocking - @ssandbekkhaug - Can you add a before and after pic of your changes in the PR description? I honestly cannot tell the difference between the screens on the Screenshot test.

Comment (swatkat1) [reply server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss:29]: nm- I see the difference.

**Review by swatkat1 — APPROVED** [review]
thanks for the work!

File: browser-test/image_snapshots/program_deep_link_test/login-prompt-for-guest-users-using-program-slug-north-star.png
Comment (jcma-google) [inline browser-test/image_snapshots/program_deep_link_test/login-prompt-for-guest-users-using-program-slug-north-star.png:1]: maybe I'm missing something but I don't see any difference in the images?

Comment (jcma-google) [reply browser-test/image_snapshots/program_deep_link_test/login-prompt-for-guest-users-using-program-slug-north-star.png:1]: oh wait I see it now!

**Pull request merged**